src/ProjectTemplates/Web.ProjectTemplates/content/StarterWeb-FSharp/Program.fs

```diff
@@ -36,7 +36,7 @@ module Program =
 
         if not (builder.Environment.IsDevelopment()) then
             app.UseExceptionHandler("/Home/Error")
-#if HasHttpsProfile
+#if (HasHttpsProfile)
```

Comment:
What about line 13?

Reply:
Do we have this issue in any other F# templates?

Reply:
We did. Thanks for mentioning it. I've fixed them up. I've also fixed up line 13.